wrappers/python/simtk/openmm/__init__.py

```diff
@@ -20,6 +20,8 @@
 
 from simtk.openmm.openmm import *
 from simtk.openmm.vec3 import Vec3
+from mtsintegrator import MTSIntegrator
+from amd import AMDIntegrator, AMDForceGroupIntegrator, DualAMDIntegrator
```

Comment:
We do this in other places and have no trouble with Python 3.  See https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/openmm/app/__init__.py.